src/main/java/net/dries007/tfc/common/blocks/crop/ClimbingCropBlock.java

```diff
@@ -99,8 +99,9 @@ public void die(Level level, BlockPos pos, BlockState state, boolean fullyGrown)
         final BlockPos posAbove = pos.above();
         final BlockState stateAbove = level.getBlockState(posAbove);
         final boolean hasTop = stateAbove.getBlock() == this;
-        final BlockState deadState = dead.get().defaultBlockState().setValue(DeadCropBlock.MATURE, fullyGrown || hasTop).setValue(STICK, state.getValue(STICK));
-        if (hasTop || (stateAbove.isAir() && state.getValue(STICK)))
+        final BlockState deadState = dead.get().defaultBlockState().setValue(DeadCropBlock.MATURE, fullyGrown).setValue(STICK, state.getValue(STICK));
+        // the dead immature version of the crop is NEVER two tall, so we cannot allow it to place the top block
```

Comment:
Are you sure? Dead crop blocks with a stick placed will still have a stick on the top block, does this account for that?